QUICKSTART-ECS.md

```diff
@@ -144,6 +144,8 @@ aws iam attach-role-policy \
    --policy-arn arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
 ```
 
+**Note:** The instance also need permission to access ECR repository where [default control container](https://github.com/bottlerocket-os/bottlerocket-control-container) live; the appropriate policy to add to INSTANCE_ROLE_NAME role is `AmazonEC2ContainerRegistryReadOnly`
```

Comment:
Since this policy is required to fetch the container to make this feature work, let's add it to the `aws iam...` command directly above.

Reply:
Actually, @samuelkarp pointed out that the ECS policy referenced in the previous step, which is a requirement before getting here, includes the read-only permissions we need for ECR.

@saigoneer, can you please confirm that you ran the previous step, and created the initial policy as per [the ECS instructions](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/instance_IAM_role.html)?

Reply:
I confirm it worked, the ecsInstanceRole actually had the policy AmazonEC2ContainerServiceforEC2Role, which covers AmazonEC2ContainerRegistryReadOnly.

Anyway, we still need to instruct somewhere about the two necessary policies for INSTANCE_ROLE_NAME: AmazonEC2ContainerServiceforEC2Role and AmazonSSMManagedInstanceCore.

@tjkirch adding another line ```--policy-arn``` with AmazonEC2ContainerServiceforEC2Role should be fine?

Reply:
> Anyway, we still need to instruct somewhere about the two necessary policies for INSTANCE_ROLE_NAME: AmazonEC2ContainerServiceforEC2Role and AmazonSSMManagedInstanceCore.

@saigoneer I'm sorry, I'm not sure I follow.  The command just above this adds AmazonSSMManagedInstanceCore, and the step before that describes following the ECS docs to set up AmazonEC2ContainerServiceforEC2Role.  I don't think we want to repeat the ECS instructions in here.  Can you please describe what's missing?

Reply:
@tjkirch Actually, the policy AmazonEC2ContainerServiceforEC2Role which need for INSTANCE_ROLE_NAME didn't mentioned in QUICKSTART, until I opened the role ```ecsInstanceRole``` in AWS.

Reply:
Yeah, it's there from following the ECS guide that we link to in the "IAM role" section above.  I don't think we don't need to call out AmazonEC2ContainerServiceforEC2Role by name here, we just need to make sure the user is set up.  We could copy some of the instructions out of the ECS guide and into this quickstart, but I'm hesitant because I think the ECS team is a better owner for that information.  Is that your concern in general, users not having to look at other resources?  I think that's going to be unavoidable, given that we have to have some other tools set up as well.  We need to strike a balance between a concise guide and a self-contained guide, and I think linking to ECS docs is a good thing here.

Reply:
> We could copy some of the instructions out of the ECS guide and into this quickstart, but I'm hesitant because I think the ECS team is a better owner for that information. Is that your concern in general, users not having to look at other resources? I think that's going to be unavoidable, given that we have to have some other tools set up as well.

@tjkirch I see, that makes sense. Thank you for explaining, I'm closing the PR.